aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -854,12 +857,35 @@ Tensor normal(double mean, double std, IntArrayRef size,
   return result.normal_(mean, std, generator);
 }
 
+Tensor normal_meta(double mean, double std, IntArrayRef size,
+                   c10::optional<Generator> generator,
+    c10::optional<ScalarType> dtype,
+    c10::optional<Layout> layout,
+    c10::optional<Device> device,
+    c10::optional<bool> pin_memory) {
+  CHECK_NORMAL_STD(std);
+  // See [Note: hacky wrapper removal for TensorOptions]
+  TensorOptions options = TensorOptions().dtype(dtype).layout(layout).device(device).pinned_memory(pin_memory);
+
+  auto result = at::empty(size, options);
+  return result;  // similar to normal_meta_
+}
+
 Tensor& normal_out(double mean, double std,
                    IntArrayRef size, c10::optional<Generator> generator, Tensor& result) {
   result.resize_(size);
   return result.normal_(mean, std, generator);
 }
 
+Tensor& normal_out_meta(double mean, double std,
+                        IntArrayRef size, c10::optional<Generator> generator, Tensor& result) {
+  CHECK_NORMAL_STD(std);
+  result.resize_(size);
```

Comment:
For consistency, I think it's better to use `at::native::resize_output`, here.

Reply:
could you explain why? i have no preference, this just copies what the impl version does above.

Reply:
https://github.com/pytorch/pytorch/blob/94abf120c82b613bcac1570b8910f8b7898885b6/aten/src/ATen/native/Resize.h#L12-L24